src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PosBlockSignatureRule.cs

```diff
@@ -86,13 +89,20 @@ private bool CheckBlockSignature(PosBlock block)
                 return false;
             }
 
-            if (ops.Count < 2) // script.GetOp(pc, opcode, vchPushValue)
+            if (ops.Count != 2)
```

Comment:
2 looks like a magic number.  Could this be a constant ?

Reply:
2 stands for 2 opcodes in total: OP_RETURN + OP_PUSH <data>

Reply:
we wanted to make super minimal changes, not beautify anything, So no changes of what was the before were done unless necessary